src/components/badge/badge.scss

```diff
@@ -0,0 +1,44 @@
+@import '../../assets/scss/variables';
+
+.badge {
+  display: inline-block;
+  padding: 0 7px;
```

Comment:
These look good!  I think I would like to see a little extra space on these like:
```
    padding: 0 9px;
    line-height: 20px;
```